ArchiSteamFarm/Bot.cs

```diff
@@ -1948,7 +1916,14 @@ internal sealed class Bot : IDisposable {
 			}
 
 			ArchiLogger.LogChatMessage(false, message, notification.chat_group_id, notification.chat_id, notification.steamid_sender);
-			await HandleMessage(notification.chat_group_id, notification.chat_id, notification.steamid_sender, message).ConfigureAwait(false);
+
+			string response = await Commands.Parse(this, notification.steamid_sender, message).ConfigureAwait(false);
+			if(response == null) {
```

Comment:
Missing space + use null or empty for strings (nearly always, unless it matters for you whether you have null or empty).